cursorless-talon-dev/src/cursorless_dev.talon

```diff
@@ -3,6 +3,8 @@ tag: user.cursorless
 
 {user.cursorless_homophone} record:
     user.run_rpc_command("cursorless.recordTestCase")
+{user.cursorless_homophone} record one:
+    user.run_rpc_command("cursorless.recordOneThenPause")
```

Comment:
I'd be even more verbose `cursorless.recordOneTestCaseThenPause`

Reply:
done everywhere but TestCaseRecorder.ts, where it felt like overkill